Translation corrections (#503)
07th-mod/higurashi-console-arcs: Update/kage_tips_133.txt

			NULL, " Anyone is free to come here.", Line_WaitForInput);
	ModPlayVoiceLS(4, 37, "ps3/s20/37/vtt_chisa3138", 256, TRUE);
	OutputLine(NULL, "でなきゃこんな汚い店、一人でなんか来るもんか」",
			NULL, " Who would else would come to this messy shop alone otherwise?\"", Line_Normal);
	ClearMessage();

//rそう、強がって返すのがいつもの口上なんだけど、…ホントの理由は、実は違う。
	if (GetGlobalFlag(GADVMode)) { OutputLineAll("", NULL, Line_ContinueAfterTyping); }
	OutputLine(NULL, "そう、強がって返すのがいつもの口上なんだけど、…ホントの理由は、実は違う。",
			NULL, "Of course, she retorted with her usual bluster... but her real intention was actually different.", Line_Normal);
	ClearMessage();

//r私はこの人といると、とことんまで甘えてしまいたくなるんだ。
	if (GetGlobalFlag(GADVMode)) { OutputLineAll("", NULL, Line_ContinueAfterTyping); }
	OutputLine(NULL, "私はこの人といると、とことんまで甘えてしまいたくなるんだ。",

> 			NULL, "When she was around him, it made her want to be dote upon others.", Line_Normal);

Reviewer: ```suggestion
			NULL, "When she was around him, it made her want to indulge in being spoiled.", Line_Normal);
```
